packages/react-emotion/src/index.js

```diff
@@ -116,12 +116,17 @@ const createStyled = (tag, options: { e: string }) => {
         )
       }
     }
+
     Styled.contextTypes = contextTypes
 
     Styled.__emotion_styles = styles
     Styled.__emotion_base = baseTag
     Styled.__emotion_real = Styled
 
+    Styled.displayName = `Styled(${typeof baseTag === 'string'
+      ? baseTag
+      : baseTag.displayName || 'Component'})`
```

Comment:
Should `baseTag.name` also be here?

Reply:
Not sure where `name` comes from.

Reply:
![screen shot 2017-09-30 at 1 31 59 pm](https://user-images.githubusercontent.com/11481355/31042097-cb574296-a5e3-11e7-8c2e-1dc0900cf622.png)